pdf2image/pdf2image.py

```diff
@@ -196,6 +207,16 @@ def convert_from_bytes(
             grayscale -> Output grayscale image(s)
     """
 
+    # We make sure that if passed arguments are Path objects, they're converted to strings
+    if isinstance(pdf_file, pathlib.PurePath):
+        pdf_file = pdf_file.as_posix()
```

Comment:
`pdf_file` is a byte string, there is no need for this check as it will never be a path.